src/lightly_train/_data/file_helpers.py

```diff
@@ -128,31 +126,18 @@ def open_image_tensor(image_path: Path) -> Tensor:
 
 def open_image_numpy(
     image_path: Path,
-    mode: Literal["RGB", "L", "UNCHANGED", "MASK"] = "RGB",
+    mode: Literal["RGB", "UNCHANGED", "MASK"] = "RGB",
```

Comment:
Have we thought about doing these things with an Enum? Something like
```python
class ImageMode(str, Enum):
    RGB = "RGB"
    UNCHANGED = "UNCHANGED"
    MASK = "MASK"
```
From Python 3.11 on there is also a `StrEnum` class.